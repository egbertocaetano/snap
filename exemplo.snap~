int main()
{
	int 4 + 6;
	2 - 3;
	1 + 1;

	a = 7 + 8;
}
