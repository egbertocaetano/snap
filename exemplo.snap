string t = "Amor";

int main()
{	

	string j = "Hamashia";

	t = j;
	write(t);	

}
